<?xml version="1.0" encoding="UTF-8"?><ui version="4.0"> <class>MainWindow</class> <widget class="QMainWindow" name="MainWindow">  <property name="geometry">   <rect>    <x>0</x>    <y>0</y>    <width>800</width>    <height>600</height>   </rect>  </property>  <property name="windowTitle">   <string>MainWindow</string>  </property>  <widget class="QWidget" name="centralwidget">   <layout class="QVBoxLayout" name="verticalLayout_2">    <item>     <widget class="QGroupBox" name="groupBox">      <property name="title">       <string>Control Panel</string>      </property>      <layout class="QVBoxLayout" name="verticalLayout">       <item>        <layout class="QHBoxLayout" name="horizontalLayout_2">         <item>          <widget class="QLabel" name="label_2">           <property name="minimumSize">            <size>             <width>70</width>             <height>0</height>            </size>           </property>           <property name="maximumSize">            <size>             <width>70</width>             <height>16777215</height>            </size>           </property>           <property name="text">            <string>address:</string>           </property>           <property name="alignment">            <set>Qt::AlignCenter</set>           </property>          </widget>         </item>         <item>          <widget class="QLineEdit" name="lineEdit_address">           <property name="sizePolicy">            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">             <horstretch>0</horstretch>             <verstretch>0</verstretch>            </sizepolicy>           </property>           <property name="minimumSize">            <size>             <width>200</width>             <height>0</height>            </size>           </property>           <property name="maximumSize">            <size>             <width>200</width>             <height>16777215</height>            </size>           </property>           <property name="text">            <string>127.0.0.1</string>           </property>          </widget>         </item>         <item>          <widget class="QLineEdit" name="lineEdit_port">           <property name="sizePolicy">            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">             <horstretch>0</horstretch>             <verstretch>0</verstretch>            </sizepolicy>           </property>           <property name="minimumSize">            <size>             <width>100</width>             <height>0</height>            </size>           </property>           <property name="maximumSize">            <size>             <width>100</width>             <height>16777215</height>            </size>           </property>           <property name="text">            <string>9090</string>           </property>          </widget>         </item>         <item>          <widget class="QPushButton" name="pushButton_connect">           <property name="text">            <string>connect</string>           </property>          </widget>         </item>         <item>          <widget class="QPushButton" name="pushButton_disconnect">           <property name="text">            <string>disconnect</string>           </property>          </widget>         </item>         <item>          <widget class="QPushButton" name="pushButton_delete">           <property name="text">            <string>DeleteClient</string>           </property>          </widget>         </item>        </layout>       </item>       <item>        <layout class="QHBoxLayout" name="horizontalLayout">         <item>          <widget class="QLabel" name="label">           <property name="minimumSize">            <size>             <width>70</width>             <height>0</height>            </size>           </property>           <property name="maximumSize">            <size>             <width>70</width>             <height>16777215</height>            </size>           </property>           <property name="text">            <string>internal:</string>           </property>           <property name="alignment">            <set>Qt::AlignCenter</set>           </property>          </widget>         </item>         <item>          <widget class="QSpinBox" name="spinBox_interval">           <property name="sizePolicy">            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">             <horstretch>0</horstretch>             <verstretch>0</verstretch>            </sizepolicy>           </property>           <property name="minimumSize">            <size>             <width>80</width>             <height>0</height>            </size>           </property>           <property name="maximumSize">            <size>             <width>80</width>             <height>16777215</height>            </size>           </property>           <property name="minimum">            <number>1</number>           </property>           <property name="maximum">            <number>2000</number>           </property>          </widget>         </item>         <item>          <widget class="QPushButton" name="pushButton_starttest">           <property name="text">            <string>start test</string>           </property>          </widget>         </item>         <item>          <widget class="QPushButton" name="pushButton_stoptest">           <property name="text">            <string>stop test</string>           </property>          </widget>         </item>         <item>          <widget class="QPushButton" name="pushButton_sendobj">           <property name="text">            <string>send obj</string>           </property>          </widget>         </item>        </layout>       </item>       <item>        <layout class="QHBoxLayout" name="horizontalLayout_3">         <item>          <widget class="QLineEdit" name="lineEdit_text"/>         </item>         <item>          <widget class="QPushButton" name="pushButton_sendtext">           <property name="text">            <string>send text</string>           </property>          </widget>         </item>        </layout>       </item>      </layout>     </widget>    </item>    <item>     <widget class="QGroupBox" name="groupBox_2">      <property name="title">       <string>Message Box</string>      </property>      <layout class="QVBoxLayout" name="verticalLayout_3">       <item>        <widget class="QPlainTextEdit" name="plainTextEdit">         <property name="maximumBlockCount">          <number>100</number>         </property>        </widget>       </item>      </layout>     </widget>    </item>   </layout>  </widget>  <widget class="QMenuBar" name="menubar">   <property name="geometry">    <rect>     <x>0</x>     <y>0</y>     <width>800</width>     <height>23</height>    </rect>   </property>  </widget>  <widget class="QStatusBar" name="statusbar"/> </widget> <resources/> <connections/></ui>